node/subsystem-util/src/nesting_sender.rs

```diff
@@ -196,10 +196,9 @@ where
 
 // Helper traits and implementations:
 
-impl<M, M1> Clone for NestingSender<M, M1>
+impl<M> Clone for NestingSender<M, M>
```

Comment:
`M, M1` was a relic from an old version of the file. I changed it to `M, M` to be consistent with the constructor.